frontend/src/metabase/admin/settings/components/SettingsPages/LocalizationSettingsPage.tsx

```diff
@@ -0,0 +1,57 @@
+import { t } from "ttag";
+import _ from "underscore";
+
+import { useSetting } from "metabase/common/hooks";
+import { Box, Flex, Stack } from "metabase/ui";
+
+import { AdminSettingInput } from "../widgets/AdminSettingInput";
+import { FormattingWidget } from "../widgets/FormattingWidget";
+
+export function LocalizationSettingsPage() {
+  const availableLocales = useSetting("available-locales");
+  const availableTimezones = useSetting("available-timezones");
+  return (
+    <Flex justify="space-between">
+      <Box w="36rem">
+        <Stack gap="xl">
+          <AdminSettingInput
+            name="site-locale"
+            title={t`Instance language`}
+            options={_.sortBy(availableLocales || [], ([, label]) => label).map(
+              ([code, label]) => ({ label, value: code }),
+            )}
+            inputType="select"
+          />
+          <AdminSettingInput
+            name="report-timezone"
+            searchable
+            title={t`Report Timezone`}
+            options={[
```

Comment:
We need to override description here, or update it on the backend to match the existing UI

Before | After
---|---
![Screenshot 2025-05-15 at 7 29 00 AM](https://github.com/user-attachments/assets/baa3bb11-c8f6-4bab-a5cd-4ab41e37b7e9) | ![Screenshot 2025-05-15 at 7 31 37 AM](https://github.com/user-attachments/assets/d836b1ca-33d8-4b11-bf68-599d5fafa2b1)